src/bci_build/package/openjdk.py

```diff
@@ -61,12 +63,16 @@ def _get_openjdk_kwargs(
     }
 
     if devel:
+        # don't set CMD in SP6 onward as jshell is broken and the CMD is
```

Comment:
SP6 is already released, did you mean SP7?

Reply:
I meant SP6, because it's already broken on SP6

Reply:
not on platforms other than ppc64le. I think we should do behavior changes only with unreleased SPs, not within the maintenance period of a released SP.